tests/pytest/test.py

```diff
@@ -251,10 +251,10 @@ def testGetWithPathErrors(env):
     # If paths contain illegal characters, the error message must not contain them
 
     # Path (and error message) with embedded nulls in path
-    r.expect('JSON.GET', 'x', 'gar\x00\x00bage').raiseError().contains("does not exist")
+    r.expect('JSON.GET', 'x', 'gar\x00\x00bage').raiseError()
```

Comment:
Why not test also some of the error message?

Reply:
Fixed